scripts/integration/docker-compose.aws.yml

```diff
@@ -0,0 +1,75 @@
+services:
+  mock-ec2-metadata:
+    image: timberiodev/mock-ec2-metadata:latest
+    networks:
+      - backend
+
+  mock-localstack:
+    image: localstack/localstack-full:0.11.6
+    environment:
+      - SERVICES=kinesis,s3,cloudwatch,elasticsearch,es,firehose,sqs
```

Comment:
NOT BLOCKING: I wonder if it's worth splitting up the aws tests _more_ to improve e2e timing if we're just testing a single service 🤔

Reply:
yeah, I thought about it. But when testing `aws` entirely it would mean starting 7 container for nothing... And the localstack thing is *just* a mock in the end...
Not sure it's worth it...